modules/ROOT/attachments/admin-api.yaml

```diff
@@ -851,6 +851,17 @@ paths:
         200:
           description: Success
           content: {}
+  /partitions/rebalance_cores:
```

Comment:
This belongs to https://github.com/redpanda-data/docs/pull/615/commits which should be merged by the GA release